ci/gem-push.yml

```diff
@@ -22,7 +20,7 @@ jobs:
     # uses: ruby/setup-ruby@v1
       uses: ruby/setup-ruby@55283cc23133118229fd3f97f9336ee23a179fcf # v1.146.0
       with:
-        ruby-version: 2.6.x
+        ruby-version: "2.6"
```

Comment:
[in my tests](https://github.com/MayCXC/Kink/actions/runs/9726785732/job/26845606433#step:3:6) `2.6.x` is not a valid input for `ruby/setup-ruby`, but `2.6` is.